tests/test_other.py

```diff
@@ -9198,3 +9198,13 @@ def test(code):
     assert lf - 900 <= f <= lf - 500
     # both is a little bigger still
     assert both - 100 <= lf <= both - 50
+
+  def test_mmap_and_munmap(self):
+    cmd = [PYTHON, EMCC, path_from_root('tests', 'mmap_and_munmap.c')]
+    for f in ['data_ro.dat', 'data_rw.dat']:
+        create_test_file(f, 'Test file')
+        cmd.extend(['--embed-file', f])
+    run_process(cmd)
+    result = run_js('a.out.js')
+    self.assertContained('failures: 0', result)
+    self.assertNotContained('FAILED', result)
```

Comment:
in `other`, it's easier to use `self.do_other_test`.

That can only check for existing text, but I think it's enough to look for `failures: 0`?

Reply:
Done.